releasenotes/notes/moved-error-constants-97c7278a381b56d0.yaml

```diff
@@ -0,0 +1,11 @@
+---
+upgrade:
+  - |
+     Instead of using error constants from ``ddtrace.ext.errors``. Use constants from ``ddtrace.constants`` module. 
+     For Example:: ``ddtrace.ext.errors.ERROR_MSG`` -> ``ddtrace.constants.ERROR_MSG``
```

Comment:
```suggestion
     For example: ``ddtrace.ext.errors.ERROR_MSG`` -> ``ddtrace.constants.ERROR_MSG``
```

Reply:
👍  will fix this with the other suggestions